### ISSUE TITLE
Different data split from the previous works

### ISSUE THREAD

**Issue by shimazing** [CVLAB-Unibo/Feature-Distillation-for-3D-UDA#2]
Thank you for releasing the code.
I noticed that you use the entire training set for model training. I think this is unfair because you use a larger dataset for training than the one used in previous baselines. They split the training data into a training set and a validation set. 

Did I miss anything?

**Comment by adricarda** [comment]
Hi,
as far as I know, I am using the same splits of previous works (at least the ones I am familiar with and that I actually run the code). 
To which baseline are you referring? 

Best

**Comment by shimazing** [comment]
For example, 

https://github.com/zou-longkun/GAST/blob/master/data/dataloader_Norm.py
https://github.com/VITA-Group/MLSP/blob/main/PointDA/data/dataloader.py

In those links, you can see that they both partition the train set into the train and the validation set.
In your case, I think you use not the the part (20%) of train set  but the source test set as the validation set.
As a result, it seems that you use more training instances.

**Comment by adricarda** [comment]
I am not aware of MLSP. Regarding GAST I think they changed the code after I opened an issue on their GitHub page. Previously, they used the target domain to obtain their best checkpoint.

**Comment by shimazing** [comment]
I already know that. 
But the point is that the difference of the training set used by all (as far as I know,,,) the baselines (80% of train split) and your implementation (100% of train split).

Could you let me know the reference that uses the same split with this work?

**Comment by adricarda** [comment]
I used PointDAN as a main reference.
I don't think that all the models use the protocol you mentioned. Anyway, if this is not detailed in the paper you cannot expect that we all do it in the same way. 
Moreover, as long as it is the source domain, I don't think that is unfair, it makes sense to use all the data.

**Comment by shimazing** [comment]
Thank you! I checked the protocol that PointDAN used.

https://github.com/IdanAchituve/DefRec_and_PCM/blob/master/PointDA/data/dataloader.py
This is another example which uses 80% of training set for training.  Furthermore, the papers after this seem to follow the protocol used in this paper (They all specify the links of their implementation or clarify whose protocol they followed in their papers) 
As long as you directly brought the results from their paper, their protocol should be followed for fair comparison.

**Comment by adricarda** [comment]
Hi, I tried a few settings training with only 80% of the source data and only for the first step of our pipeline.
In modelnet->scannet and shapenet->scannet I obtained 57.5 and 55.0 respectively, which are comparable with the results in Tab. 4 of the main paper (57.2 and 55.0). 
In any case I recommend using the whole source train set. You can then use the source test set as a validation set, and the target test set should be your test set.
